charts/featureform/values-staging.yaml

```
@@ -0,0 +1,35 @@
+psql:
+  createSecret: false
+  db: postgres
+  host: oss-staging.cfqp5tvkxko3.us-east-1.rds.amazonaws.com
+  password: null
+  port: 5432
+  sslmode: require
+  user: postgres
+
+stateProvider: psql
+
+versionOverride: 0.15.2
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)